velox/docs/functions/datetime.rst

```diff
@@ -11,6 +11,10 @@ Date and Time Functions
     Returns the UNIX timestamp ``unixtime`` as a timestamp with time zone
     using ``string`` for the time zone.
 
+.. function:: millisecond(timestamp) -> int64
```

Comment:
Generic question: How are we testing for / ensuring compatibility with the Presto functions, e.g. https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java ?